### ISSUE TITLE
Resource Access Option - Missing documentation of new option names

### ISSUE THREAD

**Issue by a82kd** [sandboxie-plus/Sandboxie#1708]
### Is your feature request related to a problem or use case?

The access options have been renamed in more recent versions (Normal, Open, Open for All, Closed, Read Only, Box Only).
However, the meaning of these options in not alway clear (e.g. Normal vs. Open?)

### Describe the solution you'd like

Please add either a description of options (tooltip?) in the software. Or alternatively please update the documentation.

**Comment by DavidXanatos** [comment]
Normal is only relevant when usign rule specificity, 
normal is the norlam behavioure of sandboxie, reading alowed but writes are virtualized.
setting a path to normal is only menainfgull when a parent path was first set to soemthign else, like done in the privacy mode.

**Comment by a82kd** [comment]
This is true for the regular sandboxes. However, if I understand your answer correctly, for a **Privacy Enhanced** sandbox the **Normal** option would be the preferred option to open directories, right?

**Comment by DavidXanatos** [comment]
yes than you allow read but still prevent write
